src/main/java/uk/gov/hmcts/ccd/v2/internal/resource/UIStartTriggerResource.java

```diff
@@ -37,6 +39,16 @@ public static UIStartTriggerResource forDraft(@NonNull CaseEventTrigger caseEven
         return new UIStartTriggerResource(caseEventTrigger, draftId, ignoreWarning, Origin.DRAFT);
     }
 
+    @JsonProperty("id")
```

Comment:
just for my info how come this change?

Reply:
Id was null because the HAL resource has it marked with the JsonIgnore. It is inconsistent with the other endpoints.